samples/excel/89-preview-apis/worksheet-tab-color.yaml

```diff
@@ -0,0 +1,110 @@
+id: excel-89-preview-apis-worksheet-tab-color-
+name: Worksheet tab color
+description: Set and get the tab color of a worksheet
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |
+        $("#set-tab-color-to-hex-color").click(() => tryCatch(setTabColorToHexColor));
+        $("#set-tab-color-to-named-color").click(() => tryCatch(setTabColorToNamedColor));
+        $("#set-tab-color-to-default-color").click(() => tryCatch(setTabColorToDefaultColor));
+        $("#get-tab-color").click(() => tryCatch(getTabColor));
+
+        async function setTabColorToHexColor() {
+            await Excel.run(async (context) => {
+                const activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.tabColor = "#FF0000";
+
+                await context.sync();
+            });
+        }
+
+        async function setTabColorToNamedColor() {
+            await Excel.run(async (context) => {
+                const activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.tabColor = "yellow";
+
+                await context.sync();
+            });
+        }
+
+        async function setTabColorToDefaultColor() {
+            await Excel.run(async (context) => {
+                const activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.tabColor = "";
+
+                await context.sync();
+            });
+        }
+
+        async function getTabColor() {
+            await Excel.run(async (context) => {
+                const activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.load("name, tabColor");
+
+                await context.sync();
+
+                let tabColor = activeSheet.tabColor !== "" ? activeSheet.tabColor : "the default color";
```

Comment:
Please reverse the !==.  A negative with an else is a double negative.  So do a comparison for `=== ""` instead.